### PR TITLE
refactor: Modify user cookie and page judge

### PR DESCRIPTION
**Pull request by whDongRui**
* feat: Developer user can view as normal

### CODE REVIEW

File: server/routes/login.js [hunk 1]
```diff
@@ -30,12 +30,11 @@ router.get('/logout', ctx => {
     httpOnly: false
   };
   ctx.session = null;
-  ctx.cookies.set('user', '', cookieOptions);
```
Comment (iwisunny) [inline server/routes/login.js]: use a forEach loop to set values

File: src/index.js [hunk 2]
```diff
@@ -20,6 +20,11 @@ if (typeof window !== 'undefined') {
   const AppWithRouter = withRouter(App);
   try {
     store.loginUser = JSON.parse(getCookie('loginUser') || '{}');
+    const changeDev = getCookie('changeDev');
```
Comment (iwisunny) [inline src/index.js]: `changeDev` is a verb, rename it as noun

File: src/locales/zh/translation.json [hunk 3]
```diff
@@ -347,5 +347,8 @@
   "Upgrade cluster": "升级集群版本",
   "Rollback": "回滚",
 
+  "Back to user": "返回普通用户",
```
Comment (iwisunny) [inline src/locales/zh/translation.json]: 切换为普通用户

File: src/locales/zh/translation.json [hunk 4]
```diff
@@ -347,5 +347,8 @@
...
+  "Back to developer": "返回开发者",
```
Comment (iwisunny) [inline src/locales/zh/translation.json]: 切换为开发者

File: src/pages/Profile/index.jsx [hunk 5]
```diff
@@ -23,9 +23,10 @@ export default class Profile extends Component {
   }
 
   componentDidMount() {
-    const { fetchDetail } = this.props.userStore;
-    const userId = getCookie('userId');
-    fetchDetail(userId);
+    const { userStore, loginUser } = this.props;
+    const { fetchDetail } = userStore;
+    const { user_id } = loginUser;
+    fetchDetail(user_id);
```
Comment (iwisunny) [inline src/pages/Profile/index.jsx]: `await fetchDetail(user_id)`

File: src/routes/index.js [hunk 6]
```diff
@@ -70,14 +70,27 @@ const routes = {
   '*': Home
 };
 
+const judgeNeedAuth = route => {
+  if (route.startsWith('/:dash')) {
+    return true;
+  }
+
+  const paths = ['/runtimes', '/purchased', '/profile', '/ssh_keys'];
+  if (paths.includes(route)) {
```
Comment (iwisunny) [inline src/routes/index.js]: simplify it: `return paths.includes(route)`

File: server/routes/login.js (hunk 1 above)
Comment (whDongRui) [reply server/routes/login.js]: ok.

File: src/index.js (hunk 2 above)
Comment (whDongRui) [reply src/index.js]: ok, rename 'role'.

File: src/locales/zh/translation.json (hunk 3 above)
Comment (whDongRui) [reply src/locales/zh/translation.json]: ok.

File: src/locales/zh/translation.json (hunk 4 above)
Comment (whDongRui) [reply src/locales/zh/translation.json]: ok.

File: src/pages/Profile/index.jsx (hunk 5 above)
Comment (whDongRui) [reply src/pages/Profile/index.jsx]: ok

File: src/routes/index.js (hunk 6 above)
Comment (whDongRui) [reply src/routes/index.js]: ok.

**Pull request merged**